From: Net-SNMP Packaging Team <pkg-net-snmp-devel@lists.alioth.debian.org>
Date: Thu, 18 Jun 2015 06:12:05 +0900
Subject: _vacm_missing_dependency_check

Do not call vacm_standard_views() if the corresponding module was not compiled.
(applied upstream in changeset 19116)

http://net-snmp.git.sourceforge.net/git/gitweb.cgi?p=net-snmp/net-snmp;a=commitdiff;h=93161743ab8caa5259a945a8d890ae96a99f0b1c;hp=2299559c29fc6493ac2f8836b82683c0904a9b9d

However, it isn't applied newest 5.7.2 source tree... why?
---
 agent/agent_read_config.c | 3 +++
 apps/snmptrapd_auth.c     | 1 +
 2 files changed, 4 insertions(+)

diff --git a/agent/agent_read_config.c b/agent/agent_read_config.c
index f9ef1af..68a3bd8 100644
--- a/agent/agent_read_config.c
+++ b/agent/agent_read_config.c
@@ -290,6 +290,9 @@ update_config(void)
     snmp_call_callbacks(SNMP_CALLBACK_APPLICATION,
                         SNMPD_CALLBACK_PRE_UPDATE_CONFIG, NULL);
     free_config();
+#ifdef USING_MIBII_VACM_CONF_MODULE
+    vacm_standard_views(0,0,NULL,NULL);
+#endif
     read_configs();
 }
 
diff --git a/apps/snmptrapd_auth.c b/apps/snmptrapd_auth.c
index be21e5e..10765ce 100644
--- a/apps/snmptrapd_auth.c
+++ b/apps/snmptrapd_auth.c
@@ -27,6 +27,7 @@
 #endif
 
 #include <net-snmp/agent/agent_trap.h>
+#include <net-snmp/agent/net-snmp-agent-includes.h>
 
 /**
  * initializes the snmptrapd authorization code registering needed
